code/hud/hudtarget.cpp

```diff
@@ -6521,11 +6522,9 @@ void HudGaugeOffscreen::calculatePosition(vertex* target_point, vec3d *tpos, vec
 	codes_or = (ubyte)(target_point->codes | eye_vertex->codes);
 	clip_line(&target_point,&eye_vertex,codes_or,0);
 
-	if (!(target_point->flags&PF_PROJECTED))
-		g3_project_vertex(target_point);
+	g3_project_vertex(target_point);
 
-	if (!(eye_vertex->flags&PF_PROJECTED))
-		g3_project_vertex(eye_vertex);
+	g3_project_vertex(eye_vertex);
```

Comment:
Also, a bit tenuously related, but the preceding `clip_line()` function will *change* the `world` position, so the point should be re-projected into `screen` no matter what! I'm fairly sure the transformation preserves its screen direction, but still.